docs/reference/packages.md

```
@@ -268,3 +268,34 @@ mylib/
+
+---
+
+## Native Function Naming Convention
+
+Stdlib package functions that are implemented as C runtime functions follow the `__ry_<package>_<function_name>` convention.
+
+> **Note**: This convention applies to stdlib package functions (e.g., `base64`, `filesystem`, `path`). Built-in functions (e.g., `print`, `length`) and math functions use varied implementations (inline LLVM IR, libc calls) and do not follow this naming pattern.
+
+### Format
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)